components/dashboard/src/components/ContextMenu.tsx

```diff
@@ -50,12 +51,16 @@ function ContextMenu(props: ContextMenuProps) {
                 {props.children}
             </div>
             {expanded?
-                <div className={`z-50 ${props.width || 'w-40'} bg-white absolute py-2 right-0 flex flex-col border border-gray-200 rounded-lg space-y-2`}>
+                <div className={`mt-2 z-50 ${props.width || 'w-40'} bg-white absolute right-0 flex flex-col border border-gray-200 rounded-lg`}>
```

Comment:
**suggestion**: Let's change the dropdown width to `w-48` as with the more actions button (see https://github.com/gitpod-io/gitpod/pull/3496#pullrequestreview-616162423).

```suggestion
                <div className={`mt-2 z-50 ${props.width || 'w-48'} bg-white absolute right-0 flex flex-col border border-gray-200 rounded-lg`}>
```